### PR TITLE
fix Hodgkin-Huxley model variable names

### PR DESCRIPTION
**Pull request by clinssen**
This PR fixes the Hodgkin-Huxley model variable names following https://github.com/nest/nest-simulator/pull/1256

### CODE REVIEW

File: models/hh_psc_alpha.nestml [hunk 1]
```diff
@@ -60,8 +60,8 @@ neuron hh_psc_alpha:
     function beta_h_init  real = 1. / ( 1. + exp( -( V_m / mV + 35. ) / 10. ) )
 
     Act_m real =  alpha_m_init / ( alpha_m_init + beta_m_init )   # Activation variable m
-    Act_h real = alpha_h_init / ( alpha_h_init + beta_h_init )    # Activation variable h
-    Inact_n real =  alpha_n_init / ( alpha_n_init + beta_n_init ) # Inactivation variable n
+    Inact_h real = alpha_h_init / ( alpha_h_init + beta_h_init )    # Activation variable h
```
Comment (Silmathoron) [inline models/hh_psc_alpha.nestml]:
```suggestion
    Inact_h real = alpha_h_init / ( alpha_h_init + beta_h_init )    # Inactivation variable h for Na
```

File: models/hh_psc_alpha.nestml [hunk 2]
```diff
@@ -60,8 +60,8 @@ neuron hh_psc_alpha:
...
+    Act_n real =  alpha_n_init / ( alpha_n_init + beta_n_init ) # Inactivation variable n
```
Comment (Silmathoron) [inline models/hh_psc_alpha.nestml]:
```suggestion
    Act_n real =  alpha_n_init / ( alpha_n_init + beta_n_init ) # Activation variable n for K
```

File: models/hh_psc_alpha.nestml [hunk 3]
```diff
@@ -60,8 +60,8 @@ neuron hh_psc_alpha:
     function beta_h_init  real = 1. / ( 1. + exp( -( V_m / mV + 35. ) / 10. ) )
 
     Act_m real =  alpha_m_init / ( alpha_m_init + beta_m_init )   # Activation variable m
```
Comment (Silmathoron) [inline models/hh_psc_alpha.nestml]:
```suggestion
    Act_m real =  alpha_m_init / ( alpha_m_init + beta_m_init )   # Activation variable m for Na
```

File: models/hh_psc_alpha.nestml [hunk 4]
```diff
@@ -201,8 +201,8 @@ neuron hh_psc_alpha_implicit:
...
```
Comment (Silmathoron) [inline models/hh_psc_alpha.nestml]:
```suggestion
    Act_m real =  alpha_m_init / ( alpha_m_init + beta_m_init )   # Activation variable m for Na
```

File: models/hh_psc_alpha.nestml [hunk 5]
```diff
@@ -201,8 +201,8 @@ neuron hh_psc_alpha_implicit:
...
-    Act_h real = alpha_h_init / ( alpha_h_init + beta_h_init )    # Activation variable h
-    Inact_n real =  alpha_n_init / ( alpha_n_init + beta_n_init ) # Inactivation variable n
+    Inact_h real = alpha_h_init / ( alpha_h_init + beta_h_init )    # Activation variable h
```
Comment (Silmathoron) [inline models/hh_psc_alpha.nestml]:
```suggestion
    Inact_h real = alpha_h_init / ( alpha_h_init + beta_h_init )    # Inactivation variable h for Na
```

File: models/hh_psc_alpha.nestml [hunk 6]
```diff
@@ -201,8 +201,8 @@ neuron hh_psc_alpha_implicit:
...
+    Act_n real =  alpha_n_init / ( alpha_n_init + beta_n_init ) # Inactivation variable n
```
Comment (Silmathoron) [inline models/hh_psc_alpha.nestml]:
```suggestion
    Act_n real =  alpha_n_init / ( alpha_n_init + beta_n_init ) # Activation variable n for K
```

**Comment by clinssen** [comment]
Thank you for the suggestions!

**Comment by jougs** [comment]
If @Silmathoron is happy, I am happy.
Merging!

**Pull request merged**